packages/libs/react-ui/src/components/Form/InputWrapper/InputWrapper.context.ts

```diff
@@ -0,0 +1,16 @@
+import type { vars } from '@theme/vars.css';
+
+import { createContext } from 'react';
+import type { Status } from '../Form.css';
+
+export type OpenSections = string[];
+
+interface IInputWrapperContext {
+  status?: Status;
```

Comment:
I saw earlier there was a `FormFieldStatus` type. Should that be used here as well or are these different?

Reply:
we do export it as FormFieldStatus, inside the context of form is called status, i can rename for consistency

Reply:
yes will do, i think it makes more sense